### PR TITLE
Edits learn.md

### PR DESCRIPTION
**Pull request by janga1997**
When I wanted to get started with Mockup, I found that the documentation in LEARN.md did not reflect the recent changes. Do suggest further changes if necessary.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ Changelog
 2.3.0 (Unreleased)
 ------------------
 
+Fixes:
+
+- Updated the documentation in LEARN.md
+  [janga1997]
+
+2.3.0 (Unreleased)
+------------------
```
Comment (gforcada) [inline CHANGES.rst]: @janga1997 this header is redundant to the one above, please remove it

**Comment by janga1997** [comment]
@gforcada I hope it's okay now?

**Comment by gforcada** [comment]
@janga1997 the README.rst part yes, as for the other part I didn't look into mockup much yet myself.

But let's ping a few ones that have been doing so: @thet @vangheem @jcbrand @frapell 

btw. a new release of mockup seems to be highly needed with the huuuge changelog that it has.

**Comment by vangheem** [comment]
Looks good. Thank you.

**Comment by janga1997** [comment]
@vangheem Is there a place where I can get the attention of the whole mockup team?

**Comment by gforcada** [comment]
@janga1997 opening issues/pull requests here or in [patternslib](https://github.com/patternslib/patterns) is probably the best way to get their attention.

Hanging on IRC or community.plone.org is another means if they are more chatty or discussion like questions that you have.

**Comment by jcbrand** [comment]
Hi @janga1997 If the patterns aren't Plone specific, then please make the pull request to Patternslib, if they're Plone-specific, then please make them here.

**Comment by janga1997** [comment]
@jcbrand I wanted to get the opinion of a few people before working rightaway on a pattern. Should I open up a issue at Patternslib, or just submit a pull request after completing work on one?

**Comment by jcbrand** [comment]
@janga1997 If you want people's opinion, then open up an issue. I read all the issues in Patternslib, not so much in Mockup, but there @vangheem and others are available.

**Pull request merged**